src/commons/history-string-builder.ts

```diff
@@ -0,0 +1,93 @@
+import { HistoryStringParts, SEPARATOR } from '../data-types/constants';
```

Comment:
Maybe SEPARATOR would fit better as a class attribute since it's specific role is specifically to do with the history